proposals/cleanup.md

```diff
@@ -46,39 +46,36 @@ There are two main use cases associated with this.
         iv. Must allow deletion to cancel the future clean-up action
         v. Must provide mechanism by which cluster operator (Kyverno CLI) can query across a cluster and return all resources which will be cleaned up
-    b. kyverno.io/expires
-        i. Must support absolute time in future in at least one standardized time format (ISO 8601 recommended)
-        ii. Must allow both assignment to new (incoming) resources as well as assignment to existing resources
-        iii. Must allow alteration after the fact
-        iv. Must allow deletion to cancel the future clean-up action
-        v. Must provide mechanism by which cluster operator (Kyverno CLI) can query across a cluster and return all resources which will be cleaned up
+        vi. Must support absolute time in future in at least one standardized time format (ISO 8601 recommended)
+        vii. Must allow both assignment to new (incoming) resources as well as assignment to existing resources
+        viii. Must allow deletion to cancel the future clean-up action
+        ix. Must allow alteration after the fact 
     c. General:
         i. Must log clean-up of any and all resources in, minimally, the Kyverno log
         ii. Must support clean-up of both Namespaced and cluster-scoped resources (including custom)
-        iii. Must only pay attention to these defined annotations (i.e., user cannot define a custom annotation as a replacement for these built-in, special Kyverno annotations)
-        iv. If both ttl and expires are specified, either Kyverno ignores the resource or it operates on the sooner of the two. Validate policy may be written which alternatively prevents this condition (account for both creation and updating a resource to add the second annotation)
+        iii. Must only pay attention to these defined labels (i.e., user cannot define a custom label as a replacement for these built-in, special Kyverno labels)
+        iv. If both ttl and expires are specified, either Kyverno ignores the resource or it operates on the sooner of the two. Validate policy may be written which alternatively prevents this condition (account for both creation and updating a resource to add the second label)
 2. Rule:
     a. Must support definition in both Policy and ClusterPolicy resources.
     b. Must support a cron-style schedule around which field validation must be placed (ex., ensuring it uses only a desired subset of the total cron functionality; ensuring it does not run too frequently; etc.)
     c. Must only work in background mode and therefore must ensure background=true for this rule type
     d. Must use existing Kyverno match/exclude and other policy authoring mechanisms to define logic
     e. Must support clean-up of both Namespaced and cluster-scoped resources (including custom)
-    f. Must only clean up resources which have neither ttl nor expires annotations assigned
+    f. Must only clean up resources which have either ttl or expires labels assigned
```

Comment:
We didn't implement this, let's clarify if we want this behaviour or not.

cc @chipzoller

Reply:
We have the same logic for ttl label? Am I right? If the ttl label is defined then only the resource will be removed otherwise there will be no action on the resource.

Reply:
Thought we don't support expires labels?

Reply:
Yes, you're right @realshuting updated the docs.